components/delete_post_modal.jsx

```diff
@@ -149,8 +146,11 @@ export default class DeletePostModal extends React.Component {
                         />
                     </button>
                     <button
-                        ref='deletePostBtn'
+                        ref={(deletePostBtn) => {
```

Comment:
Should we make this as an instance function instead of inline function like `ref={this.setInputRef}`?

Reply:
The reason I made it an inline call is because this section is rendered conditionally, and is not re-rendered often. Making it an instance function means that it is run as soon as the class is instantiated, even if this section is never rendered; adding overhead (however small) to the component. 
Inline, it is only run once the button is actually being rendered on the screen. And, because it won't be re-rendering a lot, I do not worry about it being called a lot. 
I think moving it to an instance function is a pre-mature optimization that, without testing, may not speed anything up. It may actually hurt performance.

That is my 2c. i am not tied to this opinion though. (2/5) :)